manager/src/test/java/com/redhat/service/bridge/manager/actions/connectors/ConnectorActionUtilsTest.java

```diff
@@ -0,0 +1,45 @@
+package com.redhat.service.bridge.manager.actions.connectors;
+
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import static com.redhat.service.bridge.manager.actions.connectors.ConnectorAction.PROCESSORS_PARAMETER;
+import static com.redhat.service.bridge.manager.actions.connectors.ConnectorActionUtils.LOG_PROCESSOR_MULTILINE_PARAMETER;
+import static com.redhat.service.bridge.manager.actions.connectors.ConnectorActionUtils.LOG_PROCESSOR_PARENT_PARAMETER;
+import static com.redhat.service.bridge.manager.actions.connectors.ConnectorActionUtils.LOG_PROCESSOR_SHOWHEADERS_PARAMETER;
+import static com.redhat.service.bridge.manager.actions.connectors.ConnectorActionUtils.addLogProcessorToDefinition;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ConnectorActionUtilsTest {
+
+    public static String EXPECTED_PROCESSORS_JSON = "\"" + PROCESSORS_PARAMETER + "\":[" +
+            "  {" +
+            "    \"" + LOG_PROCESSOR_PARENT_PARAMETER + "\": {" +
+            "        \"" + LOG_PROCESSOR_MULTILINE_PARAMETER + "\":true," +
+            "        \"" + LOG_PROCESSOR_SHOWHEADERS_PARAMETER + "\":true" +
+            "    }" +
+            "  }" +
+            "]";
+
+    @Test
+    void testAddLogProcessorToDefinition() throws JsonProcessingException {
+        ObjectMapper mapper = new ObjectMapper();
+        ObjectNode initialDefinition = mapper.createObjectNode();
```

Comment:
`emptyJson` instead of `initialDefinition`?

Reply:
This test was deleted with the new refactoring